inductiva/utils/files.py

```diff
@@ -100,3 +104,34 @@ def get_alphanum_key(file):
     files = sorted(files, key=get_alphanum_key)
 
     return files
+
+
+def download_from_url(url: str,
+                      local_file_path: str,
+                      save_dir: Optional[str] = None) -> str:
+    """Download a file from an URL.
+
+    Args:
+        url: The URL to download the file from.
+        local_file_path: Name attributed to the file to download.
+        save_dir: The directory to save the file to.
+    Returns:
+        The path to the downloaded file.
+    """
+
+    if save_dir is not None:
+        local_file_path = pathlib.Path(save_dir, local_file_path)
+
+    local_file_path = resolve_path(local_file_path)
+
+    if not local_file_path.parent.exists():
+        local_file_path.parent.mkdir(parents=True)
+
+    try:
+        urllib.request.urlretrieve(url, local_file_path)
+        logging.info("File downloaded to %s", local_file_path)
+    except urllib.error.URLError as url_error:
+        logging.error("Could not download file from %s", url)
+        logging.error("%s", url_error)
+
+    return str(local_file_path.absolute())
```

Comment:
Seems like this local path will be returned whether the file is download or not. Right? So it fails the download, it will return a local path to a non-existent file, no?

Reply:
Indeed! I will pass this to the try part and raise an error here.